k8s/statefulset.yaml

```diff
@@ -0,0 +1,120 @@
+---
+## ConfigMap
+#apiVersion: v1
+#kind: ConfigMap
+#metadata:
+#  name: uptime-kuma-config-map
+#data:
+#  xxx: 'xxx'
+---
+## StatefulSet
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: uptime-kuma
+spec:
+  replicas: 1
+  serviceName: uptime-kuma-service
+  selector:
+    matchLabels:
+      app: uptime-kuma
+  template:
+    metadata:
+      labels:
+        app: uptime-kuma
+    spec:
+      containers:
+      - name: uptime-kuma
+        # Choose your own mirror version
+        image: louislam/uptime-kuma:1.10.0-alpine
+        imagePullPolicy: IfNotPresent
+        ports:
+        - name: uptime-kuma
+          containerPort: 3001
+          protocol: TCP
+        resources:
+          requests:
+            cpu: 100m
+            memory: 256Mi
+          limits:
+            cpu: 500m
```

Comment:
May want to increase this, uptime-kuma is also running an internal database.

Reply:
The default resource limit and  apiVersion has been changed   4c3c16b030ba7808c346532b12193912fd289a23
```
        # Compute Resource Quota https://kubernetes.io/docs/concepts/policy/resource-quotas/#compute-resource-quota
        resources:
          # The node running Pod must meet the most basic requirements to run Pod.
          requests:
            cpu: 500m
            memory: 1024Mi
          # The maximum amount of memory and CPU that can be used during Pod operation
          limits:
            cpu: 1000m
            memory: 2048Mi
```